insights/static/js/data-display.js

```diff
@@ -368,7 +368,8 @@ var app = new Vue({
                 this.mapUrl = PAGE_URLS['map'];
                 this.dataUrl = PAGE_URLS['data'];
             }
-            history.pushState(this.filters, '', "?" + queryParams.toString());
+
+            window.location.href = window.location.pathname + '?' + queryParams.toString();
```

Comment:
Not a blocker, but would be good to know the rationale for this either with a comment or some additional information in the commit message body, as generally we prefer to avoid page reloads.

Reply:
There appears to be a bug in chartjs (or _somewhere_ at least) that gets the order of bars muddled when the chart updates without a page reload:
![Screenshot 2023-06-15 at 07-16-42 360Giving Insights](https://github.com/ThreeSixtyGiving/360insights/assets/49149591/5e8d0ab6-ad4f-4505-b8dc-c6e43422c45d)

This fixes itself when the page refresh is refreshed, so I've switched the chart refresh to a full page refresh to sidestep the issue. I suspect it'd be [resolved if we upgraded chartjs to 4.3.0](https://vue-chartjs.org/guide/#updating-charts), but that would likely be a fair bit of work.